### PR TITLE
Add smoke testing capability to CI

### PR DESCRIPTION
**Pull request by emilioward**
### What is the context of this PR?
Describe what you have changed and why, link to other PRs or Issues as appropriate.

This branch is a copy of the latest changes of the old smoketest branch. This was done due to so many commit and conflicts with the master branch when trying to rebase and squash. 

### How to review 
Describe the steps required to test the changes (include screenshots if appropriate).

Check over the code and test to see if the smoketests work locally and with travis.

### CODE REVIEW

**Comment by samiwel** [comment]
[Add smoke tests (5)](https://trello.com/c/ltEZIBdG/26-add-smoke-tests-5)

File: .travis.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ script:
   - yarn lint -- --max-warnings=0
   - yarn test -- --coverage
   - yarn build
-  - chmod +x ./scripts/run_smoketests.sh
```
Comment (samiwel) [inline .travis.yml]: For future reference, you can apply the `chmod +x` permissions to the file before committing the change to the repository. I think git will detect the permission change and honour it when you check it out.

Comment (emilioward) [inline .travis.yml]: Ok thank you

**Comment by WickyNilliams** [comment]
But [the title is hard-coded in the HTML](https://github.com/ONSdigital/eq-author/blob/da8f4dabc5c7235a439ec5b84267bb3714efb583/public/index.html#L16). So this test doesn't really prove anything 😕

**Comment by samiwel** [comment]
@WickyNilliams Agreed, but at the same time, the sooner the testing scaffolding gets merged the sooner we can all contribute to improving the test suite 👍  Perhaps it's worth just adding 1 additional test step just make sure the "Select to begin" text appears?

**Comment by WickyNilliams** [comment]
Yeah, I'd say that test is essential

**Comment by WickyNilliams** [comment]
@samiwel you're a hero getting this to work!

**Pull request merged**